### PR TITLE
Added literature for advanced edge detection techniques for underwater applications and image enhancement techniques for underwater data

### PR DESCRIPTION
**Pull request by HJGrant**

### CODE REVIEW

File: references/fishsense.bib [hunk 1]
```diff
@@ -391,6 +391,33 @@ @Article{
   groups = {Image Processing},
 }
 
+@Article{10.1109/ICOEI.2019.8862794,
+  author  = {Akshita Saini, Mantosh Biswas},
+  journal = {ICOEI},
+  title   = {Object Detection in Underwater Image by DetectingEdges using Adaptive Thresholding},
+  year    = {2019},
+  file    = {:C\:/Users/Hamish/Downloads/Object_Detection_in_Underwater_Image_by_Detecting_Edges_using_Adaptive_Thresholding.pdf:PDF},
```
Comment (ntlhui) [inline references/fishsense.bib]: @ccrutchf how do we want to handle files?

File: references/fishsense.bib [hunk 2]
```diff
@@ -391,6 +391,33 @@ @Article{
...
+  groups  = {Image Processing},
+}
+
+@Article{10.1109/ACCESS.2019.2932611,
+  author  = {Miao Yang},
+  journal = {IEEE Access},
+  title   = {An In-Depth Survey of Underwater Image Enhancement and Restoration},
+  year    = {2019},
+  file    = {:C\:/Users/Hamish/Downloads/An_In-Depth_Survey_of_Underwater_Image_Enhancement_and_Restoration.pdf:PDF},
```
Comment (ntlhui) [inline references/fishsense.bib]: Local file!

File: references/fishsense.bib [hunk 3]
```diff
@@ -391,6 +391,33 @@ @Article{
...
+  groups  = {Image Processing},
+}
+
+@Article{10.1109/ICIP.2014.7025927,
+  author  = {Fu, Xueyang and Zhuang, Peixian and Huang, Yue and Liao, Yinghao and Zhang, Xiao-Ping and Ding, Xinghao},
+  journal = {2014 IEEE International Conference on Image Processing (ICIP)},
+  title   = {A retinex-based enhancing approach for single underwater image},
+  year    = {2014},
+  file    = {:C\:/Users/Hamish/Downloads/A_retinex-based_enhancing_approach_for_single_underwater_image.pdf:PDF},
```
Comment (ntlhui) [inline references/fishsense.bib]: Local file!

**Review by ntlhui — CHANGES_REQUESTED** [review]

**Comment by HJGrant** [comment]
I've removed the lines that contained the local file paths. I had them in there so that I could find my files through the JabRef software.

**Comment by ntlhui** [comment]
@ccrutchf please establish the policy for how you want to store the PDF files

**Review by ntlhui — APPROVED** [review]

**Pull request merged**